File: Coursera - Build Your First Android App/Block08/app/src/main/java/com/example/block8/MainActivity.java
package com.example.block8;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void sendMessage(View v) {
        String message = ((EditText) (findViewById(R.id.editText_message))).getText().toString();
        Uri destination = Uri.parse("smsto:5556");
        Intent smsIntense = new Intent(Intent.ACTION_SENDTO, destination);
        smsIntense.putExtra("sms_body", message);
        startActivity(smsIntense);
        Log.i("TAG_I_made_up", "I'm here!");

    }
}

File: firstApp - animasi 1/app/src/main/java/com/example/firstapp/MainActivity.kt
package com.example.firstapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var garutIsShowing = true

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun garutOnClick(view: View) {
        Log.i("info", "garut telah di klik")

        var garut = garutImageView
        var nanas = nanasImageView

        if(garutIsShowing) {
            garut.animate().alpha(0f).setDuration(2000)
            nanas.animate().alpha(1f).setDuration(2000)
            garutIsShowing = false
        } else {
            garut.animate().alpha(1f).setDuration(2000)
            nanas.animate().alpha(0f).setDuration(2000)
            garutIsShowing = true
        }
    }
}

